doc/manual/import.rst

```diff
@@ -10,7 +10,7 @@ in your `.netrc`_ file. You need to install `httpie`_ and replace the
 ``<API_URL>`` in the examples below with the API URL of your local DOMjudge
 installation.
 
-For using the CLI, you need to replace ``<WEBAPP_DIR>`` with the path to
+To use the CLI, you need to replace ``<WEBAPP_DIR>`` with the path to
```

Comment:
I might be wrong, but I think we don't mention `<WEBAPP_DIR>` anymore after your commit.

So I propose something like:
```suggestion
To use the CLI, you need to run inside the ``webapp`` directory of the DOMserver.
```

If I understand how the current CLI is implemented.

Reply:
Yea so we drop this whole line imho

Reply:
Ah yes you are right. Sorry, this uses console except for in `Putting all API imports together`